src/migration/1726691862710-User.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class User1726691862710 implements MigrationInterface {
+  name = 'User1726691862710'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "user" ADD "followingEmail" boolean NOT NULL DEFAULT true`);
```

Comment:
Just thinking is there any case where we want to consider false now?
I do think when someone turned off all emails for instance or all push this should be false... Maybe something to consider in the migration (should be a easy query)

From FE it should work the same.
Turning the main email off should turn off all sub emails immediatly.

CC: @capJavert fyi

Reply:
I agree, we can do once of UPDATE query or script but we need to check with product because they usually don't like emails off 🙈

Reply:
@capJavert do we only want to check for the Activity and Community updates, or also personalized digest?
Because when email notifications are "Off", all of them are off. But if you enable digest, it technically back on.

Reply:
I think the first one is fine for now

Reply:
For emails it's as simple as
```sql
UPDATE
	"user"
SET
	"followingEmail" = FALSE
WHERE
	"user"."acceptedMarketing" = FALSE
	AND "user"."notificationEmail" = FALSE;
```

It would update 120k rows

Reply:
For push notifications it's either on or it's not. So there we could just keep it on as default

Reply:
We should probably run it manually in batches. Once when this is merged, and once just as we go live to catch any changes that might've been done while UI wasn't in place

Reply:
Anything else missing from the PR?

Reply:
Easier to run a query to turn it off for 120k users, then the other 870k that has either enabled